subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -54,6 +54,12 @@ enum {
 			       1),
 #endif /* CONFIG_BT_CONN */
 
+#if defined(CONFIG_BT_CTLR_USER_EXT)
+	TICKER_ID_USER_BASE,
+	TICKER_ID_USER_LAST = (TICKER_ID_USER_BASE +
+			       CONFIG_BT_CTLR_USER_TICKER_ID_RANGE - 1),
+#endif /* CONFIG_BT_CTLR_USER_EXT */
+
```

Comment:
Doesn't this block only make sense if CONFIG_BT_CTLR_USER_TICKER_ID_RANGE > 0?

Reply:
> Doesn't this block only make sense if CONFIG_BT_CTLR_USER_TICKER_ID_RANGE > 0?

Correct. Should we add the condition CONFIG_BT_CTLR_USER_TICKER_ID_RANGE > 0 to line 57?

Reply:
> > Doesn't this block only make sense if CONFIG_BT_CTLR_USER_TICKER_ID_RANGE > 0?
> 
> Correct. Should we add the condition CONFIG_BT_CTLR_USER_TICKER_ID_RANGE > 0 to line 57?

That would make sense.